packages/theme-language-server-common/src/documentLinks/DocumentLinksProvider.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('DocumentLinksProvider', () => {
     const uriString = 'file:///path/to/non-liquid-html-document.txt';
     const rootUri = 'file:///path/to/project';
 
-    documentManager.open(URI.parse(uriString).toString(), 'Sample plain text content', 1);
+    documentManager.open(path.normalize(uriString), 'Sample plain text content', 1);
```

Comment:
You're probably right. The normalization happens because of Windows URIs with the `\` in them. URI.parse().toString() without the `true` encodes the `\` as a `%` encoded string and it makes the URIs unreadable. Plus any logic that would have used the replace `\` by `/` would not work. 

This diff happened because I was looking for all the URI...toString() in the code and was replacing it with normalize instead.